README.md

```diff
@@ -862,3 +862,12 @@ To change the user directory and/or account name follow the instructions [here](
 [4]: https://github.com/nvm-sh/nvm/releases/tag/v0.37.2
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+**Homebrew makes zsh directories unsecure**
+
+```shell
+zsh compinit: insecure directories, run compaudit for list.
+Ignore insecure directories and continue [y] or abort compinit [n]? y
+```
+
+Homebrew causes insecure directories like `/usr/local/share/zsh/site-functions` and `/usr/local/share/zsh`. This is not really a problem of `nvm` and much more related to homebrew. Refer [here](https://github.com/zsh-users/zsh-completions/issues/680) for some solutions related to the issue.
```

Comment:
```suggestion
Homebrew causes insecure directories like `/usr/local/share/zsh/site-functions` and `/usr/local/share/zsh`. This is not an `nvm` problem - it is a homebrew problem. Refer [here](https://github.com/zsh-users/zsh-completions/issues/680) for some solutions related to the issue.
```

Reply:
@ljharb Updated that part with the changes so that it becomes more clear.